### DISCUSSION TITLE
'Updates' Dashboard: Add 'Day of week' column

### DISCUSSION THREAD

**Discussion by KrzysztofHajdamowicz** [teslamate-org/teslamate#3926] [Ideas]
Having a dozen of updates installed thorough vehicle lifetime, seeing a correlation between day of week and day of update starts to become more effort-intensive.
Having column showing day of week (Mon, Tue, Wed etc) will ease seeing a pattern.

**Comment by DrMichael** [comment]
Column should be easy... What about this?
![image](https://github.com/teslamate-org/teslamate/assets/8087682/9590f879-fe96-4dba-8312-f8ed3ff609a9)

**Reply by KrzysztofHajdamowicz** [reply]
I like that!

**Comment by DrMichael** [comment]
@JakobLichterfeld this might be a corner case... What do you think, shall I make that change in the Updates dashboard?

**Reply by cwanja** [reply]
I am not sure I see the value. Tesla is not known to release versions on a certain day. It ships when it ships and not all vehicles get the updates at the same time. Thus I am not sure the value. 

But I also do not speak for the entire project.

**Reply by JakobLichterfeld** [reply]
I can echo cwanja here, but if the info is useful for someone, it does not hurt to add. But it can raise questions we are not able to answer, as tesla backend decides on unknown criteria which vehicle get which update when (rollout by MCU, region, number of fleet updated already, if there is an open service ticket, HW version, time of engaged autopilot, …)

**Reply by DrMichael** [reply]
Well, we could just add a Weekday column. That would help @KrzysztofHajdamowicz also...

**Reply by cwanja** [reply]
@KrzysztofHajdamowicz can you explain a bit more what you are hoping to gain from seeing this data? What correlations or analysis are you trying to infer by knowing the day of the week the install was done? Is that even useful, because the day you _install_ might not be the day the software was released - maybe you were not home or missed a notification, etc.
